packages/react-components/theme-designer/src/components/Content/Content.tsx

```diff
@@ -13,13 +17,34 @@ const useStyles = makeStyles({
     justifyContent: 'center',
     flexDirection: 'column',
     ...shorthands.padding('40px', '10%', '0px', '10%'),
+    gridRowGap: tokens.spacingVerticalXXXL,
   },
 });
 
+export const brandWeb: BrandVariants = {
```

Comment:
lets make a comment that this data is temporary and will eventually be pulled from current theme